### PR TITLE
Fix warning from incorrect SimpleSelect usage in tests of DataTable component

### PR DESCRIPTION
**Pull request by imransl**
## Description
This happened due to using `defaultValue` to specify the placeholder in `SimpleSelect` as React doesn't like this. This PR tries to come with a workaround for this by using the `value` to either specify the selected value or the placeholder (if no option is selected).

## Related JIRA Issue(s)
[ENSWBSITES-1828](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1828)

## Deployment URL(s)
http://fix-simple-select.review.ensembl.org

## Views affected
BLAST settings
Genome browser -> Navigation panel -> Go to new location

### CODE REVIEW

**Comment by imransl** [comment]
There was another error that popped up due to `value` being specified without `onChange` in `SimpleSelect`. The fix for this seems a bit hacky, but couldn't think of anything better.

Screenshot 2023-02-23 at 20 57 49

File: src/shared/components/simple-select/SimpleSelect.tsx [hunk 1]
```diff
@@ -127,7 +154,8 @@ const SimpleSelect = (
       <select
         ref={selectRef}
         className={styles.selectResetDefaults}
-        defaultValue={placeholder ? '' : undefined}
+        value={getSelectValue()}
+        onChange={selectProps.onChange ?? noop}
```
Comment (azangru) [inline src/shared/components/simple-select/SimpleSelect.tsx:158]: Why do you need this line (as well as the noop)?

File: src/shared/components/simple-select/SimpleSelect.tsx [hunk 2]
```diff
@@ -40,6 +41,7 @@ type HTMLSelectProps = SelectHTMLAttributes<HTMLSelectElement>;
 export type Option = {
   value: string;
   label: string;
+  isSelected?: boolean;
```
Comment (azangru) [inline src/shared/components/simple-select/SimpleSelect.tsx:44]: Why is this needed? This adds confusion to the component's api, by introducing competition between `option.isSelected` and `props.value` (and by providing to limit to how many options can have the `isSelected` flag on them).

File: src/shared/components/simple-select/SimpleSelect.tsx (hunk 1 above)
Comment (imransl) [reply src/shared/components/simple-select/SimpleSelect.tsx:158]: I've mentioned the reason in the above comment

**Review by azangru — APPROVED** [review]
